pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/DeclarativeLinterCommandTest.java

```diff
@@ -29,6 +29,7 @@
 import hudson.model.User;
 import jenkins.model.Jenkins;
 import org.apache.commons.io.FileUtils;
+import org.jenkinsci.plugins.pipeline.modeldefinition.cli.DeclarativeLinterCommand;
```

Comment:
Maybe move the actual test into the new package as well.

Reply:
Good call.